packages/service-utils/src/lib.rs

```diff
@@ -50,11 +55,13 @@ impl ServiceAccountType {
     }
 }
 
+// Private enum mimicking the ExecuteMsg from the base_account contract
 #[cw_serde]
-pub enum ExecuteMsg {
+enum ExecuteMsg {
     ExecuteMsg { msgs: Vec<CosmosMsg> }, // Execute any CosmosMsg (approved services or admin)
 }
```

Comment:
I don't think mocking is good here, very error prone, if base account changes its api, will be hard to debug.
Maybe its good for now, not sure how accounts will be played in the future, but will be safer to use the execute enum directly from the account contract here.

Reply:
Yeah, I am not really satisfied with this solution either actually.

Reply:
I moved the `valence-base-account` 's `msg` & `error` to a new `valence-account-utils` package.